pkg/controller/tas/topology_ungater.go

```diff
@@ -102,7 +102,10 @@ func (r *topologyUngater) setupWithManager(mgr ctrl.Manager, cfg *configapi.Conf
 		)).
 		For(&kueue.Workload{}).
```

Comment:
This controller is already using `For`, is the explicit setting of MaxConcurrentReconciles still required in this case?

Reply:
I see, so can we revert that change for the controller? Also, could we use For for other controllers then to avoid the  lengthy setting of MaxConcurrentReconciles?

Reply:
If using `For`, we will lose the Generics capabilities which means we need to inference the typed everywhere.